src/csv-file-validator.js

```diff
@@ -79,9 +78,11 @@
 						if (valueConfig.name !== columnValue) {
 							file.inValidMessages.push(
 								_isFunction(valueConfig.headerError)
-									? valueConfig.headerError(columnValue, valueConfig.name, rowIndex + 1, columnIndex + 1)
-									: 'Header name ' + columnValue + ' is not correct or missing in the ' + (rowIndex + 1) + ' row / '
-										+ (columnIndex + 1) + ' column. The Header name should be ' + valueConfig.name
+									? valueConfig.headerError(
+										columnValue, valueConfig.name, rowIndex + 1, columnIndex + 1
+									)
+									: `Header name ${columnValue} is not correct or missing in the ${rowIndex + 1} row/
```

Comment:
@shystruk Are the whitespace changes here (before/after the slash) necessary changes? They're breaking some of my tests. 😅

Reply:
@camerondunford small updates on the error messages. Hope you don't have a lot of test cases to update ;)

Reply:
Thanks. It seems like an inconsistent (before/after the slash) & excessive (after the slash) application of whitespace.


![Default (-zsh) 2021-12-07 15-40-47](https://user-images.githubusercontent.com/840612/145122971-eecaf1d3-81b8-4b05-b7f9-ed48f808784a.png)

Reply:
oh, looks not right. Will fix that. Thanks!

Reply:
@camerondunford go and upgrade to 1.13.1

Reply:
Right on, thank you! 🙇